codex-rs/core/prompt.md

```diff
@@ -97,6 +97,7 @@ You can invoke apply_patch like:
 shell {"command":["apply_patch","*** Begin Patch\n*** Add File: hello.txt\n+Hello, world!\n*** End Patch\n"]}
 ```
 
+<!-- PLAN_TOOL:START -->
```

Comment:
We could also make it additive instead of subtractive if you want only one marker?